hw1/reverse.js

```diff
@@ -0,0 +1,5 @@
+const currentMaxValue = 4589;
+const reverseMaxValue = +`${currentMaxValue}`.split('').reverse().join('');
```

Comment:
Все працює вірно, по умові. Але тут використовується не явне перетворення. В абсолютній більшості випадків краще використовувати явне перетворення. Тобто в даному випадку 
```suggestion
const reverseMaxValue = Number(currentMaxValue.toString().split("").reverse().join(""));
```